.gitignore

```diff
@@ -11,5 +11,4 @@
 # Output of the go coverage tool, specifically when used with LiteIDE
 *.out
 
-# Dependency directories (remove the comment below to include it)
-# vendor/
+./main
```

Comment:
now its in artifacts/node-proof-of-existence

Reply:
What an eye! :) changed in d5b5a8b